src/components/timeline/TrackContextMenu.js

```diff
@@ -516,13 +517,46 @@ class TimelineTrackContextMenu extends PureComponent<Props> {
     );
   }
 
+  renderHideAllTracksByType() {
+    const { rightClickedTrack, globalTracks, localTracksByPid } = this.props;
+    if (rightClickedTrack === null) {
+      return null;
+    }
+    const trackIndex = rightClickedTrack.trackIndex;
+    if (rightClickedTrack.type === 'global') {
+      return (
+        <MenuItem
+          key={trackIndex + 'hide-all'}
+          preventClose={false}
+          data={(rightClickedTrack, globalTracks, localTracksByPid)}
+          onClick={this._toggleGlobalTrackVisibility}
+        >
+          Hide all {`"${this.getRightClickedTrackName(rightClickedTrack)}"`}{' '}
```

Comment:
We shouldn't use `getRightClickedTrackName` here, because we want to hide the items by their `type`. We should create a function like `getRightClickedTrackType` and use it here. So when we right click on the "Web Content" track, it should write "process" because its type is process. See: 
https://github.com/firefox-devtools/profiler/blob/0d98b6b6bd0e82d03560f64d49cee276da6d021f/src/types/profile-derived.js#L205-L217